### DISCUSSION TITLE
How can i set the default value in livewire ?

### DISCUSSION THREAD

**Discussion by dev-dhull** [livewire/livewire#5378] [2. Show and tell]
i have a multiple product when i get the default value of that product, but when i get that value in input field the default value doesn't show ?

Code :->
<x-filament::page>
    <form wire:submit.prevent="submit">
        <div>
            @if (session()->has('message'))
                <div class="alert alert-success">
                    {{ session('message') }}
                </div>
            @endif
        </div>
        <select wire:model="user_type">
            <option value="">Select User Type.....</option>
            <option value="admin">Admin</option>
            <option value="distributor">Distributor</option>
            <option value="retailer">Retailer</option>
        </select><br><br>
        <input wire:model="name" type="text" placeholder="Name"><br><br>
        <input wire:model="email" type="email" placeholder="Email Address"><br><br>
        <input wire:model="password" type="password" placeholder="Password"><br><br>

        <div align = "center">Products</div>

       ** @foreach($product as $val)
            <br><br> {{ $val->name }} :
            <input type="number" wire:model="default_price" value = "{{$val->default_price}}">
        @endforeach<br><br>**
        <button type="submit">
            Submit
        </button>
    </form>
</x-filament::page>

**Comment by joshhanley** [comment]
@dev-dhull there seems to be an issue with your pasted code. You should paste the code using code blocks.

When you paste code into github, you need to surround it in triple back ticks and then specify the file type like php or blade.

For example
\```php
\\ PHP code here
\```

\```blade
\\ HTML and blade code here
\```

Can you fix that?

Thanks!
